docs/config.md

```diff
@@ -54,13 +55,14 @@ for more details.
-| `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | *                  | source               |
-| `NetRuntime` | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime)                                                                                   | *                  | source               |
-| `Process`    | [OpenTelemetry.Instrumentation.Process](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)                                                                                   | *                  | source               |
+| ID            | Instrumented library                                                                                                                                                                            | Supported versions | Instrumentation type |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------------------|
+| `AspNet`      | ASP.NET Framework \[1\]                                                                                                                                                                         | *                  | source               |
+| `AspNet`      | ASP.NET Core \[2\]                                                                                                                                                                              | *                  | source               |
+| `HttpClient`  | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | *                  | source               |
+| `NetRuntime`  | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime)                                                                                   | *                  | source               |
+| `Process`     | [OpenTelemetry.Instrumentation.Process](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)                                                                                   | *                  | source               |
+| `NServiceBus` | [NServiceBus](https://www.nuget.org/packages/NServiceBus)                                                                                                                                       | ≥8.0.0             | source & bytecode    |
```

Comment:
isn't `NServiceBus` metrics instrumentation not enabling traces instrumentation and vice-verse? Meaning that they are always "together"? E.g. What happens if one would set `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS=NServiceBus` ?

Can we clarify the behavior using a footer?

Reply:
Partially. NServiceBus will produce both metrics and traces together, but we are registering it in Tracer/Metrics builder separately.

It can work independently.